deployments/polygon/usdc/migrations/1675200105_raise_polygon_supply_caps.ts

```diff
@@ -51,11 +55,53 @@ export default migration('1675200105_raise_polygon_supply_caps', {
       ]
     );
 
+    const RootChainManager = await deploymentManager.existing(
+      'RootChainManager',
+      RootChainManagerAddress
+    );
+    const USDCAmountToBridge = exp(100_000, 6);
+    const COMPAmountToBridge = exp(10_000, 18);
+    const depositUSDCData = utils.defaultAbiCoder.encode(['uint256'], [USDCAmountToBridge]);
+    const depositForUSDCCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'address', 'bytes'],
+      [comet.address, USDC.address, depositUSDCData]
+    );
+    const depositCOMPData = utils.defaultAbiCoder.encode(['uint256'], [COMPAmountToBridge]);
+    const depositForCOMPCCalldata = utils.defaultAbiCoder.encode(
+      ['address', 'address', 'bytes'],
+      [rewards.address, COMP.address, depositCOMPData]
+    );
+
     const mainnetActions = [
+      // 1. Set Comet configuration and deployAndUpgradeTo new Comet on Polygon.
       {
         contract: fxRoot,
         signature: 'sendMessageToChild(address,bytes)',
         args: [bridgeReceiver.address, l2ProposalData]
+      },
+      // 2. Approve Polygon's ERC20Predicate to take Timelock's USDC (for bridging)
+      {
+        contract: USDC,
+        signature: 'approve(address,uint256)',
+        args: [ERC20PredicateAddress, USDCAmountToBridge]
+      },
+      // 3. Bridge USDC from mainnet to Polygon Comet using RootChainManager
+      {
+        target: RootChainManager.address,
+        signature: 'depositFor(address,address,bytes)',
+        calldata: depositForUSDCCalldata
+      },
+      // 4. Approve Polygon's ERC20Predicate to take Timelock's COMP (for bridging)
+      {
+        contract: COMP,
+        signature: 'approve(address,uint256)',
+        args: [ERC20PredicateAddress, COMPAmountToBridge]
+      },
+      // 5. Bridge COMP from mainnet to Polygon CometRewards using RootChainManager
+      {
+        target: RootChainManager.address,
+        signature: 'depositFor(address,address,bytes)',
+        calldata: depositForCOMPCCalldata
```

Comment:
Flows documented in Polygon docs: https://wiki.polygon.technology/docs/develop/ethereum-polygon/pos/calling-contracts/erc20/#high-level-flow

Reply:
💪 Definitely agree we want to test this on Goerli/Mumbai